lib/extensions/src/lib/config/extension.config.ts

```diff
@@ -27,9 +27,9 @@ export interface ExtensionRef {
     $license: string;
     $runtime?: string;
     $description?: string;
-
     $dependencies?: Array<string>;
 
+    appConfig?: any;
```

Comment:
Shold we update the json schema for the extension files?

Reply:
@pmartinezga we will have to maintain the whole app.config schema inside the extension schema I'm afraid

Reply:
won't it be enough by adding the appConfig attribute as an object? Something like adding here a piece of json like this:
```
 "$appConfig": {
      "description": "Application configuration parameters to add/override",
      "type": "object"
}
```

Reply:
updated schema, thanks!